portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,14 @@ function addRandomFunFacts() {
   const funFactContainer = document.getElementById('funFacts-container');
   funFactContainer.innerText = funFact;
 }
+
+async function showRandomGreets() {
```

Comment:
Same here, I thought we merged this as PR #5?

Reply:
For some reason, the servlet wasn't showing up in the sderacyDevelopment branch, so I just added the file to this git push. I think it might've been because I assumed the branch and main were at the same point since I merged them and I don't think they were. However, I do believe the branch is at the right point now. I'm gonna double-check before I do anything else